resources/autoscaling.rb

```diff
@@ -0,0 +1,169 @@
+property :return_info, [String, Array]
+property :should_decrement_desired_capacity,	[true, false], default: true
+property :asg_name,              String
+property :status_code,           String
+property :max_size,              Integer, default: 4
+
+# authentication
+property :aws_access_key,        String
+property :aws_secret_access_key, String
+property :aws_session_token,     String
+property :aws_assume_role_arn,   String
+property :aws_role_session_name, String
+property :region,                String, default: lazy { fallback_region }
+
+include AwsCookbook::Ec2 # needed for aws_region helper
```

Comment:
This comment should actually read for the fallback_region helper

Reply:
done.